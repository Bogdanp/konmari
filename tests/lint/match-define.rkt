#lang racket/base

(require racket/match)

(struct foo (x y))
(define f (foo 1 2))
(match-define (foo a b) f)
