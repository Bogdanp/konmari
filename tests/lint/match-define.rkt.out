/match-define.rkt:7:20:warning:identifier 'a' is never used
/match-define.rkt:7:22:warning:identifier 'b' is never used
